api_generator/lib/api_generator.rb

```diff
@@ -37,6 +38,20 @@ def generate(gem_folder = '../', version: nil, namespace: nil, actions: nil)
     IndexGenerator.new(gem_folder.join('lib/opensearch'), namespaces).generate
   end
 
+  def generate_low_level(gem_folder = '../')
+    gem_folder = Pathname gem_folder
+    namespaces = existing_namespaces(gem_folder)
+    low_level_namespace = 'http'
+
+    NamespaceGenerator.new(gem_folder.join('lib/opensearch/api/namespace'), low_level_namespace).generate(namespaces)
+    LowLevelBaseActionGenerator.new(gem_folder.join('lib/opensearch/api/actions'), low_level_namespace).generate
+    IndexGenerator.new(gem_folder.join('lib/opensearch'), namespaces).generate
+
+    %w[head get post put patch delete options trace connect].each do |action|
+      LowLevelActionGenerator.new(gem_folder.join('lib/opensearch/api/actions'), low_level_namespace, action).generate
+    end
```

Comment:
Should be able to do this kind of stuff inside a template, or by creating template files that are called `head.mustache`.

Reply:
as in `head.mustache`, `get.mustache`? Would that defeat the purpose of a template if we must have 1 for each method?

Reply:
This is the template being used to generate those methods: https://github.com/opensearch-project/opensearch-ruby/pull/211/files#diff-50c305414229967005b04e4f27bc2c06476c8372872a2bfaad317e9c0eb33fe0

Reply:
I generally prefer to avoid writing generator code (the loop over verbs), and would make a file that includes a generic implementation with the method as an argument, but I'm being super nitpicky, ignore me!